DEVELOPMENT.md

```diff
@@ -151,6 +151,84 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+```bash
+$ sudo dnf install postgresql
+$ psql --version
+psql (PostgreSQL) 15.6
+```
+* A SQL client is recommended to interact easily with the database. For example [DBeaver Community](https://dbeaver.io/).
+
+### Create a Table and Populate the Database
+
+The script `database/init_db.go` will create a table in the database and will insert the test objects from `database/test_objects.sql`
+* Add the database access configuration. The values are the same from the `ps-secret.yaml` or the ones
+  that you have defined in the `values.yaml`.
+```go
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = // the POSTGRES_USER from ps-secret.yaml, ie: "admin", "ps_user".
+	password = // the POSTGRES_PASSWORD from ps-secret.yaml
+	dbname   = // the POSTGRES_DB from ps-secret.yaml, ie: "postgres", "test_db".
+)
+```
+* Run the script
+```bash
+$ go run init_db.go
+table test_objects created in the DB.
+testdata from test_objects.sql inserted in the DB.
+```
+### SQL Queries (Examples)
+These queries can be executed from the database.
+Connect to the database.
+```bash
+$ kubectl exec -it [postgres_pod] -n [namespace] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+```
+```sql
+SELECT * FROM test_objects;
+```
+```sql
+SELECT * FROM test_objects WHERE kind='Job';
+```
+```sql
+SELECT * FROM test_objects WHERE data::jsonb->>'kind'='Job';
+```
+### Configure DBeaver (Optional)
+
+* Identify the PostgreSQL service.
+```bash
+$ kubectl get services -n [namespace]
+```
+* Create a port-forward in a new terminal tab.
+```bash
+$ kubectl port-forward -n [namespace] svc/postgres 5432:5432
+```
+* DBeaver settings
+  
+  Go to `Database` > `New Database Connection`.
+  Select `PostgreSQL`.
+
+  Fill the connection parameters with the values from `ps-secret.yaml`. 
+  
+  Click on the `Finish` button.
+
+  ![connection parameters](database/images/dbeaver-config.png)
+
+  Click right on the new database connection and click on `Connect`. 
+  
+  The check symbol will change from grey to green to indicate that the client is connected to the database.
+
+  ![connect to the database](database/images/connect_to_db_ok.png)
+
+  Click on the drop down to navigate the database.
+  
+  ![explore the data](database/images/explore_data.png)
+
```

Comment:
If this is optional, it does not belong to the development document. We need to keep this document as short and accessible as possible.

Reply:
Ok, I will create another document in the database folder then and I'll move this part.

Reply:
Moved it to database/README.md